lib/node/runner/node_runner.go

```diff
@@ -38,7 +38,7 @@ var DefaultHandleTransactionCheckerFuncs = []common.CheckerFunc{
 var DefaultHandleBaseBallotCheckerFuncs = []common.CheckerFunc{
 	BallotUnmarshal,
 	BallotNotFromKnownValidators,
-	BallotSaveNodeHeight,
+	BallotCheckSYNC,
```

Comment:
@Charleslee522 If `BallotCheckSYNC` checks `isBallotAcceptYes()` and only available for `ACCEPT` ballot, this, `BallotCheckSYNC` should be moved under `DefaultHandleACCEPTBallotCheckerFuncs`, is it right?

Reply:
@spikeekips It should be here because unmatched ballot is filtered in isAvailableRound method in `BallotAlreadyFinished`.

Reply:
@Charleslee522 I see. :)